server/controllers/visitation.controllers.js

```diff
@@ -0,0 +1,43 @@
+import { scheduleVisitation, getAllVisitations } from '../services/visitation.service';
+import EMAIL_CONTENT from '../../mailer';
+import { sendMail } from '../services/mailer.service';
+import httpStatus from '../helpers/httpStatus';
+
+const BALLERS_EMAIL = process.env.BALLERS_EMAIL || 'dev@highrachy.com';
+
+const VisitationController = {
+  book(req, res, next) {
+    const booking = req.locals;
+    const { user } = req;
+    scheduleVisitation({ ...booking, userId: user._id })
+      .then((schedule) => {
+        const contentBottom = `
+          <strong> Name: </strong>: ${schedule.visitorName}<br>, 
+          <strong> Phone: </strong> ${schedule.visitorPhone}<br>.
+          <strong> Email: </strong> ${
+            schedule.visitorEmail ? schedule.visitorEmail : 'no email provided'
+          } <br>.
+        `;
```

Comment:
Here is how the mail template looks like if you are interested. You will also need to set up MAILTRAP API variables in your `.env` so that your mails can be sent to MAILTRAP.

<img width="369" alt="Screenshot 2020-08-07 at 06 57 51" src="https://user-images.githubusercontent.com/26963369/89613959-552fe280-d87b-11ea-888b-7a3f9a51be49.png">


<img width="378" alt="Screenshot 2020-08-07 at 06 59 07" src="https://user-images.githubusercontent.com/26963369/89614061-8c9e8f00-d87b-11ea-9c52-0dcccb1f13e2.png">

```suggestion
        const contentBottom = `
          <strong> Name: </strong>: ${schedule.visitorName}<br /> 
          <strong> Phone: </strong> ${schedule.visitorPhone}<br />
          <strong> Email: </strong> ${
            schedule.visitorEmail ? schedule.visitorEmail : 'No email provided'
          } <br />
        `;
```